### ISSUE TITLE
New UA Query Explorer - Can't select goal metrics (metadata missing?)

### ISSUE THREAD

**Issue by TimCrank** [googleanalytics/ga-dev-tools#655]
**Describe the bug**

1. Go to 'https://ga-dev-tools.web.app/query-explorer/'
2. Provide your account,property,view and start/end dates
3. Try to select a goal based metric e.g. ga:goal1Completions

**Expected behavior**
Available goals defined for the view should be available in the metric selection drop down box or at least allow us to enter it in manually. It just shows the basic templated name of ga:goalXXCompletions but will not accept any actual metric names.

**Additional context**
Basically it appears that none of the metadata is no longer being utilised by the tool e.g. the results now use the api name for the metric e.g. ga:sessions rather than the display name (as it used to). You can't select goal metrics, page categorisation dimensions etc.
